code/modules/mob/living/carbon/human/species_types/podpeople.dm

```diff
@@ -3,7 +3,9 @@
 	name = "Anthromorphic Plant"
 	id = "pod"
 	default_color = "59CE00"
-	species_traits = list(MUTCOLORS,EYECOLOR)
+	species_traits = list(EYECOLOR,HAIR,FACEHAIR,LIPS,MUTCOLORS)
+	mutant_bodyparts = list("mam_tail", "mam_ears", "mam_body_markings", "mam_snouts", "taur", "legs")
+	default_features = list("mcolor" = "FFF","mcolor2" = "FFF","mcolor3" = "FFF", "mam_snouts" = "Husky", "mam_tail" = "Husky", "mam_ears" = "Husky", "mam_body_markings" = "Husky", "taur" = "None", "legs" = "Normal Legs")
```

Comment:
I'm very very very very very very against this. Like, can't you just man up and leave the standard one looking like a walking broccoli? It's the iconical side-effect of replica pods after all.
Bug the xenobiologist for some mutation toxin or something.